nntrainer/layers/acti_func.h

```diff
@@ -78,15 +78,18 @@ class ActiFunc {
       in_place = false;
       this->setActivation<Tensor>(gelu<T>, geluPrime<T>);
       break;
+    case ActivationType::ACT_ELU:
```

Comment:
Do you have any reason to move this code block? It is not recommended to add unrelated code changes in a commit. It makes harder to track changes and bugs in the future.

Reply:
@myungjoo Just for the code readability and uniformity(?).
All activation types except for `ACT_ELU` were placed on top of `ACT_NONE` in the switch-case. So, I moved the block to that place for some kind of uniformity(I wonder this kind of thing can be also said as unifomity...). But if you think it doesn't need to, then I'll rollback this change.

Reply:
Uh.. then never mind. It's good to go.